packages/react-storage/src/components/StorageBrowser/providers/configuration/useGetActionInputCallback.ts

```diff
@@ -22,21 +24,25 @@ export function useGetActionInputCallback({
   const [{ location }] = useStore();
   const { current, key } = location;
 
-  return React.useCallback(() => {
-    assertLocationData(current, ERROR_MESSAGE);
+  return React.useCallback(
+    (location?: LocationData) => {
+      const _location = current ?? location;
```

Comment:
Annoying, but can we add an unit test for this change? Branches is where we are having a hard time maintaining coverage